.github/workflows/php-coding-standards.yml

```diff
@@ -62,6 +62,4 @@ jobs:
 
             - name: Run PHPCS on all changed files
               run: |
-                git fetch origin trunk
-                changes=$(git diff --name-only origin/trunk..HEAD | tr '\n' ' ')
-                phpcs ${changes} -q --report=checkstyle | cs2pr
+                phpcs ./src -q --report=checkstyle | cs2pr
```

Comment:
I'm curious, previous we used `-n` which I can find documentation for but I can't seem to find any to explain `-q` ?

Reply:
It's to hide the progress while running: ` -q    Quiet mode; disables progress and verbose output`

Reply:
Thanks!